pyramid_swagger/api.py

```diff
@@ -101,26 +101,206 @@ def view_for_api_declaration(request):
     return view_for_api_declaration
 
 
-def resolve_ref(spec, url):
+def _low_level_translate(path, is_marshaling=True):
```

Comment:
With a general name like this, I think a clear docstring including precisely what type of object this returns and what the function does will be extra important.